dimagi/pages/templates/pages/toolkits/view_all.html

```diff
@@ -22,36 +19,67 @@
 {% endblock %}
 
 {% block hero %}
-  <div class="orange-theme">
-    <section class="hero hero-toolkits"
-             data-lazybg="true"
-             id="hero">
-      <div class="content content-width-large inverted align-center display-flex flex-valign-middle expand-vertical offset-top-small flex-align-center">
-        <div class="copy expand position-relative">
-
-          <h1>
-            {% blocktrans %}
-              Toolkits
-            {% endblocktrans %}
-          </h1>
-
-          <div class="headline content-width-small">
-            {% blocktrans %}
-              Drawing from 10+ years of experience, Dimagi has developed
-              several resources to support users in implementing digital
-              solutions. Download the toolkits below to help you plan and
-              implement your project.
-            {% endblocktrans %}
-          </div>{# .headline #}
-
-        </div>{# .copy #}
-      </div>{# .content #}
-    </section>
-  </div>
+  <div id="top"></div>
+  <section class="hero"
+           data-lazybg="true"
+           id="hero">
+    <div class="content align-left-at-medium align-center gap-top-large gap-top-small-at-medium gap-top-line-at-large">
+      <h2 class="gap-vert-line">
+        {% blocktrans %}
+          Toolkits
+        {% endblocktrans %}
+      </h2>
+      <h1>
+        {% blocktrans %}
+          10+ years of experience in implementing digital solutions.
+        {% endblocktrans %}
+      </h1>
+      <a class="button button-accent-gray {% click_hero 'overview' %}" 
+         href="#overview">
+        {% blocktrans %}
+          Learn More
+        {% endblocktrans %}
+      </a>
+    </div> {# .content #}
+  </section>
 {% endblock %}
 
 {% block content %}
   <section class="toolkits-container orange-theme">
     {% include 'sections/toolkits/list.html' %}
+    <div class="request-demo align-center border-top accent-gray-bg" id="request-demo">
+      <div class="content pad-vert-large content-width-medium">
+
+        <h2>
+          {% blocktrans %}
+            Trust Our Expertise
+          {% endblocktrans %}
+        </h2>
+        <p>
+          {% blocktrans %}
+            More than 2000+ projects have relied on Dimagi's technology, CommCare, 
+            for their mobile data collection and reporting needs.
+          {% endblocktrans %}
+        </p>
+
+        <div class="request-demo-form" id="hubspot-request-demo-form"></div>
+
+      </div>{# .content #}
+    </div>{# .request-demo #}
   </section>
 {% endblock %}
+
+{% block js %}
+  <!--[if lte IE 8]>
+    <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2-legacy.js"></script>
+  <![endif]-->
+  <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2.js"></script>
```

Comment:
@biyeun We have tried creating one separate similar file like `demoForm.js` and called it `inlineDemoForm.js` but we are unable to render the form and also lil bit confused how to use that file to render a form in any HTML file, if you can guide through us it will be great.